packages/d3fc-webgl/src/series/glArea.js

```diff
@@ -0,0 +1,163 @@
+import attributeBuilder from '../buffers/attributeBuilder';
+import glScaleBase from '../scale/glScaleBase';
+import programBuilder from '../program/programBuilder';
+import drawModes from '../program/drawModes';
+import areaShader from '../shaders/area/shader';
+import { rebind } from '@d3fc/d3fc-rebind';
+
+export default () => {
+  let program = programBuilder();
+  let xScale = glScaleBase();
+  let yScale = glScaleBase();
+  let decorate = () => {};
+
+  const xValueAttrib = 'aXValue';
+  const xPreviousValueAttrib = 'aXPrevValue';
+  const yValueAttrib = 'aYValue';
+  const yPreviousValueAttrib = 'aYPrevValue';
+  const y0ValueAttrib = 'aY0Value';
+  const y0PreviousValueAttrib = 'aY0PrevValue';
+  const cornerValueAttrib = 'aCorner';
+  const definedAttrib = 'aDefined';
+
+  const verticesPerElement = 6;
+
+  const draw = (numElements) => {
+    const shaderBuilder = areaShader();
+    program.vertexShader(shaderBuilder.vertex())
+      .fragmentShader(shaderBuilder.fragment())
+      .mode(drawModes.TRIANGLES);
+
+    xScale.coordinate(0);
+    xScale(program);
+    yScale.coordinate(1);
+    yScale(program);
+
+    initCornerArray(numElements);
```

Comment:
I thought we did this in `xValues` elsewhere?

Reply:
I had this separate while adding the functionality and forgot to move it